test/gtest/uct/uct_test.cc

```diff
@@ -327,6 +327,14 @@ unsigned uct_test::progress() const {
 void uct_test::flush(ucs_time_t deadline) const {
 
     bool flushed;
+#if HAVE_DC_DV
+    bool dc_mlx5 = GetParam()->tl_name.compare("dc_mlx5") == 0;
+
+    if (dc_mlx5) {
+        deadline = ucs_get_time() + ucs_time_from_sec(5);
```

Comment:
BTW why is the deadline argument passed to this function is not good enough, and needs to be changed?

Reply:
so maybe change the test which passes MAX_ULONG?